test/src/graph/VelocityDistortionTest.cpp

```diff
@@ -43,38 +45,52 @@ TEST_F(VelocityDistortionTest, smoke_test)
 		return linearVelocity.value[1] * rayTimeOffsetToPrevious * 0.001 * index > cubeSize / 2.0f;
 	};
 
+	std::vector<rgl_field_t> outFields{XYZ_VEC3_F32, IS_HIT_I32};
+
 	// Create nodes
 	EXPECT_RGL_SUCCESS(rgl_node_rays_from_mat3x4f(&useRays, rays.data(), rayCount));
 	EXPECT_RGL_SUCCESS(rgl_node_rays_set_time_offsets(&offsetsNode, timeOffsets.data(), rayCount));
-	EXPECT_RGL_SUCCESS(rgl_node_raytrace_with_distortion(&raytrace, nullptr, &linearVelocity, &angularVelocity));
+	EXPECT_RGL_SUCCESS(rgl_node_raytrace(&raytrace, nullptr));
+	EXPECT_RGL_SUCCESS(rgl_node_raytrace_configure_velocity(raytrace, &linearVelocity, &angularVelocity));
+	EXPECT_RGL_SUCCESS(rgl_node_raytrace_configure_distortion(raytrace, true));
+	EXPECT_RGL_SUCCESS(rgl_node_raytrace_configure_non_hits(raytrace, INF, INF));
```

Comment:
There is no need to configure non-hits here. Also, the configuration is the same as the default one.